src/hepattn/models/loss.py

```diff
@@ -194,36 +215,53 @@ def mask_focal_cost(pred_logits, targets, gamma=2.0, input_pad_mask=None, sample
         return torch.einsum("bnc,bmc->bnm", focal_pos, targets) + torch.einsum("bnc,bmc->bnm", focal_neg, (1 - targets))
 
 
-def mask_bce_loss(pred_logits, targets, object_valid_mask=None, input_pad_mask=None, sample_weight=None):
-    """Compute the binary cross-entropy loss for binary masks.
+def mask_bce_loss(
+    pred_logits: torch.Tensor,
+    targets: torch.Tensor,
+    object_valid_mask: torch.Tensor | None = None,  # [batch_size, num_objects]
+    input_pad_mask: torch.Tensor | None = None,  # [batch_size, num_constituents]
+    sample_weight: torch.Tensor | None = None,  # broadcastable to [batch_size, num_objects, num_constituents]
```

Comment:
I think best to leave this in the docstring, we might add autodocs at some point and don't want this going out of sync